zenodo/modules/communities/utils.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Utilities for Zenodo deposit."""
+
+from __future__ import absolute_import, unicode_literals
+
+def can_create_community():
+    """Checks it the user can create community."""
+    current_date = datetime.now()
+    confirmed_date = getattr(current_user, 'confirmed_at', None)
+    # TODO Add this variable in config
+    community_user_confirmed_since = \
+        current_app.config['COMMUNITIES_USER_CONFIRMED_SINCE']
```

Comment:
This check can go, so that we can have a unified set of "rules" as with the deposit check in `is_user_verified`